package/MDAnalysis/analysis/dssp.py

```diff
@@ -0,0 +1,279 @@
+from .base import AnalysisBase
+import numpy as np
+from einops import repeat, rearrange
+from .. import Universe
+
+CONST_Q1Q2 = 0.084
+CONST_F = 332
+DEFAULT_CUTOFF = -0.5
+DEFAULT_MARGIN = 1.0
+
+# code of the off-class functions
+# is borrowed from https://github.com/ShintaroMinami/PyDSSP
+# in accordance with the MIT License
+
+
+def _unfold(a: np.ndarray, window: int, axis: int):
+    """Unfolds an ndarray if it was batched (pydssp legacy)
+
+    Args:
+        a (np.ndarray): input array
+        window (int): window
+        axis (int): axis to unfold across
+
+    Returns:
+        np.ndarray: unfolded array
```

Comment:
Normally I'd say to reformat using numpy-style doc strings. However, given that these are private and won't have generated docs, I'll leave it as optional.

Reply:
Done, now numpy